create_bq_views.py

```diff
@@ -41,17 +41,8 @@ def get_data_view(client: bigquery.Client, data_type: str, dataset: bigquery.Dat
             {POPMAX_SQL}
             
            FROM `{dataset.project}.{dataset.dataset_id}.{data_type}_variants` as v
-    LEFT JOIN (
-        SELECT {",".join([f"gt.{f}" for f in genotypes_cols])}, 
```

Comment:
Because `s` is in both `gt` and `meta`, it needs to be prefixed in the `SELECT`. One possible easy solution would be to prefix all fields in the `SELECT`.

Reply:
Does this mean we will have to reload all tables with the prefix in the column names since the select will be looking for meta_*, v_*, and gt_*?

Reply:
Not sure if this comment still needs a response. But anyhow, we just want to prefix fields with their table name in the `SELECT`, like `gt.s`; not `gt_s`. This won't rename any fields, but resolves the ambiguity created by the `JOIN` stmts.